frontend/src/scenes/data-management/events/DefinitionHeader.tsx

```diff
@@ -25,51 +17,50 @@ export function getPropertyDefinitionIcon(definition: PropertyDefinition): JSX.E
     if (KEY_MAPPING.event[definition.name]) {
         return (
             <Tooltip title="PostHog event property">
-                <VerifiedPropertyIcon className="taxonomy-icon taxonomy-icon-verified" />
+                <IconList className="taxonomy-icon taxonomy-icon-muted" />
             </Tooltip>
         )
     }
     if (definition.verified) {
         return (
             <Tooltip title="Verified event property">
-                <VerifiedPropertyIcon className="taxonomy-icon taxonomy-icon-verified" />
+                <IconList className="taxonomy-icon taxonomy-icon-muted" />
             </Tooltip>
         )
     }
     return (
         <Tooltip title="Event property">
-            <PropertyIcon className="taxonomy-icon taxonomy-icon-muted" />
+            <IconList className="taxonomy-icon taxonomy-icon-muted" />
         </Tooltip>
     )
 }
 
 export function getEventDefinitionIcon(definition: EventDefinition & { value: string | null }): JSX.Element {
     // Rest are events
     if (definition.name === '$pageview' || definition.name === '$screen') {
-        return (
-            <Tooltip title="PostHog event">
```

Comment:
Intentionally removing the tooltip here?

Reply:
Nope!